### DISCUSSION TITLE
How to change emailVisibility

### DISCUSSION THREAD

**Discussion by edikurniawan-dev** [pocketbase/pocketbase#2791] [Q&A] [answered]
I want to list users like name, username, email etc in table, but why data i receive emailVisibility is false?
How to get email all users?

**Comment by ganigeorgiev** [comment, marked as answer]
`emailVisibility` is a system auth record field that gives the owner of the auth record (and admins) control over whether their email should be public or not. You can change the value for each user by sending an update users request with `emailVisibility: true` or update the _"Admin UI > {auth recod} > Email field > Public: On"_ toggle.

If you want to ignore the above behavior and always return the users email, You can create a helper [View collection](https://pocketbase.io/docs/collections/#view-collection) with a SQL query like:

```sql
select
    users.id,
    users.email,
    -- ... list the user fields you want to be always available
from users
```
_Note that View collections have their own List and View API rules._

**Reply by edikurniawan-dev** [reply]
thanks @ganigeorgiev
